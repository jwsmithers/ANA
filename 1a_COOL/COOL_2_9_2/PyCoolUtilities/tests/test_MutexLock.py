import unittest, sys, os

from PyCool import cool
from PyCoolCopy import PyCoolCopy, silent, dbExists
from PyCoolPayloader import PyCoolPayloader

silent()

class TestPyCoolCopy( unittest.TestCase ):

    def setUp(self):
        print 'Entering setUp'
        self.source = PyCoolPayloader( sourceConnectString,
                                       recreateDb = True )
        self.target = PyCoolPayloader( targetConnectString,
                                       recreateDb = True )
        print 'Exiting setUp'

    def tearDown(self):
        print 'Entering tearDown'
        del self.source.db
        del self.target.db
        print 'Exiting tearDown'

    def test_dbExists(self):
        print 'Entering test_dbExists'
        self.assert_( dbExists( targetConnectString ),
                      'db does not exist when it should' )
        dbSvc = cool.DatabaseSvcFactory.databaseService()
        dbSvc.dropDatabase( targetConnectString )
        print 'About to open non-existent database'
        try:
            dbSvc.openDatabase( targetConnectString )
            print '??? Database open ???'
        except Exception, e:
            print 'Exception: database not open'
            pass
        #print 'About to test existence of non-existent database'
        #self.assert_( not dbExists( targetConnectString ),
        #              'db exists when it should not' )
        print 'Exiting test_dbExists'

#######################################################################

sourceKey = "COOLTESTDB_SOURCE"
targetKey = "COOLTESTDB_TARGET"

if __name__ == '__main__':
    if sourceKey in os.environ and targetKey in os.environ:
        print 'Env variable %s and %s are both set'%( sourceKey, targetKey )
        sourceConnectString = os.environ[sourceKey]
        targetConnectString = os.environ[targetKey]
    else:
        print 'Env variable %s or %s is not set'%( sourceKey, targetKey )
        print 'Using default source and target connection strings'
        sourceConnectString = 'sqlite://;schema=sqliteTestMutexLock-S.db;dbname=COOLTEST'
        targetConnectString = 'sqlite://;schema=sqliteTestMutexLock-T.db;dbname=COOLTEST'
    print 'Source:', sourceConnectString
    print 'Target:', targetConnectString
    unittest.main( testRunner =
                   unittest.TextTestRunner(stream=sys.stdout,verbosity=2) )


