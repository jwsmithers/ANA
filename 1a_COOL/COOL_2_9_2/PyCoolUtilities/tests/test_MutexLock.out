Env variable COOLTESTDB_SOURCE or COOLTESTDB_TARGET is not set
Using default source and target connection strings
Source: sqlite://;schema=sqliteTestMutexLock-S.db;dbname=COOLTEST
Target: sqlite://;schema=sqliteTestMutexLock-T.db;dbname=COOLTEST
Entering setUp
Property Catalogue: level[Info] Adding property: OutputLevel = 5 in scope "SEAL/Services/MessageService"
Property Catalogue: level[Info] Adding property: Format =  "%-36s %-8s %s" in scope "SEAL/Services/MessageService"
Property Catalogue: level[Info] Adding property: AuthenticationFile =  "/afs/cern.ch/user/a/avalassi/private/authentication.xml" in scope "CORAL/Services/XMLAuthenticationService"
free(): invalid pointer 0xb5ab60!

python (pid=19755 ppid=7945) received fatal signal 2 (Interrupt)
signal context:
  signo  = 2, errno = 0, code = 128 (kernel)
  pid    = 0, uid = 0
  value  = (3, 0x3)
  stack  = (2, 0, (nil))

  eip: 0023:0081b939           eflags: 00000206
  eax: fffffffc   ebx: 0ac5cb64   ecx: 00000000   edx: 00000002
  esi: 00000000   edi: 002b64f2   ebp: bfff2468   esp: bfff2450
   ds: 002b        es: 002b        fs: 0000        ss: 002b

  signal esp: bfff2450  trap: 0/0  oldmask: 00000000   cr2: 00000000

  FPU:  control = ffff037f
        status  = ffff0020
        tag     = ffffffff
        ip      = 0023:028ec199
        data    = 002b:bfff1bc8
        state   = 00000020
    %fp0 = [0000:0000000000000000]
    %fp1 = [0000:0000000000000000]
    %fp2 = [0000:0000000000000000]
    %fp3 = [4002:000000000000a000]
    %fp4 = [3fff:0000000000008000]
    %fp5 = [3fff:0000000000008000]
    %fp6 = [4147:70003cf97514ea15]
    %fp7 = [4012:000000000000f424]

stack trace:
 0x04cc45f4 _ZN4seal9DebugAids10stacktraceEi + 0x78 [/afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealBase.so]
 0x04cfe7b2 _ZN4seal6Signal9fatalDumpEiP7siginfoPv + 0x200 [/afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealBase.so]
 0x04cfdf85 _ZN4seal6Signal5fatalEiP7siginfoPv + 0x1bd [/afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealBase.so]
 0x0081cf80 ? + 0x81cf80 [/lib/tls/libpthread.so.0]
 0x01c64430 _ZN5boost6detail17lightweight_mutex11scoped_lockC1ERS1_ + 0x28 [/afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so]
 0x01c6416a _ZN5boost6detail15sp_counted_base7releaseEv + 0x26 [/afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so]
 0x01c63dc8 _ZN5boost6detail12shared_countD1Ev + 0x28 [/afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so]
 0x01c63ea0 _ZN5boost10shared_ptrIN4cool9IDatabaseEED1Ev + 0x22 [/afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so]
 0x01c86eb2 ? + 0x1c86eb2 [/afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so]
 0x01157a33 _ZN6TClass10DestructorEPvb + 0x65 [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libCore.so]
 0x002b63b5 _ZN6PyROOT17op_dealloc_nofreeEPNS_11ObjectProxyE + 0x49 [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so]
 0x002b6510 _ZN6PyROOT59_GLOBAL__N__ZN6PyROOT17op_dealloc_nofreeEPNS_11ObjectProxyE10op_deallocEPNS_11ObjectProxyE + 0x1e [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so]
 0x00cedbab ? + 0xcedbab [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x002b17fd _ZN6PyROOT13TMethodHolder7ExecuteEPv + 0x139 [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so]
 0x002b1a54 _ZN6PyROOT13TMethodHolderclEPNS_11ObjectProxyEP7_objectS4_ + 0x1da [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so]
 0x002b3759 _ZN6PyROOT43_GLOBAL__N_pyroot_src_MethodProxy.cxxWTv6ob7mp_callEPNS_11MethodProxyEP7_objectS4_ + 0x6b [/afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14a90 ? + 0xd14a90 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d144dc ? + 0xd144dc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d149b4 ? + 0xd149b4 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d1454d ? + 0xd1454d [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d149b4 ? + 0xd149b4 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d1454d ? + 0xd1454d [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14c78 ? + 0xd14c78 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12a64 PyEval_EvalFrame + 0x26fc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cc047e ? + 0xcc047e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cf722b ? + 0xcf722b [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14a90 ? + 0xd14a90 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d144dc ? + 0xd144dc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14c78 ? + 0xd14c78 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12a64 PyEval_EvalFrame + 0x26fc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cc047e ? + 0xcc047e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cf722b ? + 0xcf722b [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14a90 ? + 0xd14a90 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d144dc ? + 0xd144dc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14c78 ? + 0xd14c78 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12a64 PyEval_EvalFrame + 0x26fc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cc047e ? + 0xcc047e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cf722b ? + 0xcf722b [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14a90 ? + 0xd14a90 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d144dc ? + 0xd144dc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d149b4 ? + 0xd149b4 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d1454d ? + 0xd1454d [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d149b4 ? + 0xd149b4 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d1454d ? + 0xd1454d [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cce20e ? + 0xcce20e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cc047e ? + 0xcc047e [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cf7957 ? + 0xcf7957 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cf2059 ? + 0xcf2059 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00cb7c60 PyObject_Call + 0x28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d14a90 ? + 0xd14a90 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d144dc ? + 0xd144dc [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d12697 PyEval_EvalFrame + 0x232f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d13129 PyEval_EvalCodeEx + 0x415 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d10363 PyEval_EvalCode + 0x2f [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d3fa28 ? + 0xd3fa28 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d3efd5 PyRun_SimpleFileExFlags + 0x1b1 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d3ea31 PyRun_AnyFileExFlags + 0x71 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x00d47373 Py_Main + 0x5c7 [/afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0]
 0x080486ba main + 0x22 [python]
 0x0012679a __libc_start_main + 0xda [/lib/tls/libc.so.6]
 0x0804860d Py_Main + 0x31 [python]

shared libraries present:
 0x00000000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/bin/python
 0x00c95000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/libpython2.4.so.1.0
 0x00812000 /lib/tls/libpthread.so.0
 0x00f23000 /lib/libdl.so.2
 0x00851000 /lib/libutil.so.1
 0x004ea000 /usr/lib/libstdc++.so.5
 0x0078f000 /lib/tls/libm.so.6
 0x00756000 /lib/libgcc_s.so.1
 0x00111000 /lib/tls/libc.so.6
 0x0088c000 /lib/ld-linux.so.2
 0x007d1000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/python2.4/lib-dynload/time.so
 0x0045d000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/python2.4/lib-dynload/strop.so
 0x004ab000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/python2.4/lib-dynload/readline.so
 0x00e53000 /usr/lib/libreadline.so.4
 0x00e81000 /usr/lib/libncurses.so.5
 0x0024a000 /usr/lib/libgpm.so.1
 0x0024f000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPyROOT.so
 0x00f26000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libCore.so
 0x008a4000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libCint.so
 0x08d21000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libHist.so
 0x07900000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGraf.so
 0x0059d000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGraf3d.so
 0x00b61000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGpad.so
 0x02371000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libTree.so
 0x0593d000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libMatrix.so
 0x003d8000 /lib/libcrypt.so.1
 0x00405000 /lib/libnss_files.so.2
 0x0067e000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGX11.so
 0x00411000 /usr/X11R6/lib/libXpm.so.4
 0x080da000 /usr/X11R6/lib/libX11.so.6
 0x00da4000 /usr/X11R6/lib/libXext.so.6
 0x016de000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGui.so
 0x00421000 /usr/X11R6/lib/libXcursor.so.1
 0x0042a000 /usr/X11R6/lib/libXrender.so.1
 0x00432000 /usr/X11R6/lib/X11/locale/lib/common/xlcDef.so.2
 0x00435000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libGX11TTF.so
 0x03e2c000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/cint/stl/vector.dll
 0x006cd000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libvectorDict.so
 0x0044f000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/cint/include/stdfunc.dll
 0x00db2000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libMinuit.so
 0x00ec1000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libPostscript.so
 0x00463000 /afs/cern.ch/sw/lcg/external/root/5.08.00/slc3_ia32_gcc323/root/lib/libHtml.so
 0x0049f000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/python2.4/lib-dynload/collections.so
 0x0441f000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_Cintex.so
 0x03c13000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_Reflex.so
 0x01b95000 /afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_PyCoolDict.so
 0x04c7a000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealBase.so
 0x004af000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_CoralBase.so
 0x03176000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_PluginManager.so
 0x007d6000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealKernel.so
 0x00854000 /afs/cern.ch/sw/lcg/app/releases/POOL/POOL_2_2_7/slc3_ia32_gcc323_dbg/lib/liblcg_AttributeList.so
 0x00722000 /afs/cern.ch/sw/lcg/app/releases/POOL/POOL_2_2_7/slc3_ia32_gcc323_dbg/lib/liblcg_POOLCore.so
 0x0073d000 /afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_CoolApplication.so
 0x0770d000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_Reflection.so
 0x084d0000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealServices.so
 0x0045a000 /afs/cern.ch/sw/lcg/external/uuid/1.38/slc3_ia32_gcc323/lib/libuuid.so.1
 0x00748000 /afs/cern.ch/sw/lcg/external/pcre/4.4/slc3_ia32_gcc323/lib/libpcre.so.0
 0x0075f000 /lib/libnsl.so.1
 0x01ce1000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/libSealSTLDict.so
 0x007b1000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_ReflectionBuilder.so
 0x004a3000 /afs/cern.ch/sw/lcg/external/Python/2.4.2/slc3_ia32_gcc323/lib/python2.4/lib-dynload/cStringIO.so
 0x06cbe000 /afs/cern.ch/sw/lcg/app/releases/COOL/internal/avalassi/COOL_HEAD/slc3_ia32_gcc323_dbg/lib/liblcg_RelationalCool.so
 0x00822000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_RelationalAccess.so
 0x00774000 /afs/cern.ch/sw/lcg/app/releases/SEAL/SEAL_1_7_9/slc3_ia32_gcc323_dbg/lib/liblcg_SealUtil.so
 0x00c5a000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_RelationalService.so
 0x00e3a000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_CoralCommon.so
 0x05657000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_XMLAuthenticationService.so
 0x01f1d000 /afs/cern.ch/sw/lcg/external/XercesC/2.3.0/slc3_ia32_gcc323/lib/libxerces-c.so.23
 0x02202000 /afs/cern.ch/sw/lcg/app/releases/CORAL/CORAL_1_2_1/slc3_ia32_gcc323_dbg/lib/liblcg_coral_SQLiteAccess.so
 0x028ae000 /afs/cern.ch/sw/lcg/external/sqlite/3.2.8/slc3_ia32_gcc323/lib/libsqlite3.so.0
